omnibus/config/software/oc_id.rb

```diff
@@ -21,6 +21,7 @@
 license "Apache-2.0"
 license_file "LICENSE"
 
+dependency "nokogiri" # for
```

Comment:
I also think adding nokogiri here means we'll probably end up with two versions of it? Since it's getting installed in the bundle install with the bundle-defined version, and the software definition has another version (latest on rubygems?).

On the other hand, [copying those here](https://github.com/chef/omnibus-software/blob/master/config/software/nokogiri.rb#L27-L30) might be more ugly than just installing nokogiri twice 🤔